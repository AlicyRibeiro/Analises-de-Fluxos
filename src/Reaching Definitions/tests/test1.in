3 3
a = b + c
d = a + 1
f = d + e
0
0
0