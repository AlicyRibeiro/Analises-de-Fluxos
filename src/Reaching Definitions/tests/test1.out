Bloco 0:
  IN : { }
  OUT: { (a,0,0) }
Bloco 1:
  IN : { }
  OUT: { (d,1,0) }
Bloco 2:
  IN : { }
  OUT: { (f,2,0) }
